src/JUnit.Xml.TestLogger/JUnitXmlTestLogger.cs

```diff
@@ -1,5 +1,5 @@
-﻿// Copyright (c) Spekt Contributors. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+﻿// Copyright (c) Spekt Contributors. All rights reserved. Licensed under the MIT license. See
```

Comment:
nit: new line is expected here. See https://github.com/spekt/junit.testlogger/blob/master/scripts/stylecop.json#L6

(need to find why stylecop didn't catch this)

Reply:
I'm sure the reason it changed is b/c I started running code maid, which is re-flowing comments to be a consistent length.